plugins/woocommerce/includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php

```diff
@@ -410,6 +410,16 @@ private function get_paypal_create_order_request_params( $order, $payment_source
 		$payee_email         = sanitize_email( (string) $this->gateway->get_option( 'email' ) );
 		$shipping_preference = $this->get_paypal_shipping_preference( $order );
 
+		$src_locale = get_locale();
+		// If the locale is longer than PayPal's string limit (10).
+		if ( strlen( $src_locale ) > WC_Gateway_Paypal_Constants::PAYPAL_LOCALE_MAX_LENGTH ) {
+			// Keep only the main language and region parts.
+			$locale_parts = explode( '_', $src_locale );
+			if ( count( $locale_parts ) >= 2 ) {
```

Comment:
Nit: It feels like we could limit this to `>` - if we only have two elements in the array, we don't need to put them back together again.
```suggestion
			if ( count( $locale_parts ) > 2 ) {
```

Reply:
Makes sense. I applied this fix in https://github.com/woocommerce/woocommerce/pull/61678 (commit https://github.com/woocommerce/woocommerce/pull/61678/commits/d0cdb099f805e889c21ffb1fddfa731b049d37a6)